test/cmd/create.sh

```diff
@@ -100,17 +100,7 @@ os::cmd::expect_success "_output/oshinko delete klondike3"
 
 os::cmd::expect_failure_and_text "_output/oshinko create klondike4 --metrics=notgonnadoit" "must be 'true', 'false', 'jolokia', or 'prometheus'"
 
-# exposeui
```

Comment:
I think these uses of "get" to confirm the create can use the -d flag and be put back